sktime/forecasting/xLSTM.py

```diff
@@ -0,0 +1,523 @@
+# !/usr/bin/env python3 -u
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements xLSMT forecaster.
+
+uses the existing code from https://github.com/muslehal/xLSTMTime.
+"""
+
+__all__ = ["xLSTM"]
+__author__ = ["muslehal", "vedantag17"]
+
+import numpy as np
+import pandas as pd
+import torch
+import torch.nn as nn
```

Comment:
please issues soft dependencies - see here 
https://www.sktime.net/en/latest/developer_guide/dependencies.html

Reply:
as @jgyasu suggests, I would vendor the original package and move it to `libs`, with soft dependencies isolated. See how `_safe_import` is used throughout.